### PR TITLE
LPS-141062 search-experiences-web: Fix apply baseline contributors functionality

### PR DESCRIPTION
**Pull request by thektan**
https://issues.liferay.com/browse/LPS-141062

Property rename.

cc: @oliv-yu

### CODE REVIEW

**Comment by thektan** [comment]
ci:reopen

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/710073534e...9679e2a3d8

File: modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/utils/data.js [hunk 1]
```diff
@@ -144,3 +104,43 @@ export const BASELINE_CLAUSE_CONTRIBUTORS_CONFIGURATION = {
...
+};
+
+export const DEFAULT_ADVANCED_CONFIGURATION = {};
+
+export const DEFAULT_BASELINE_SXP_ELEMENTS = [
+	boostAllKeywordsMatch,
+
+	// TODO Update once all elements are available
+
+	// TEXT_MATCH_OVER_MULTIPLE_FIELDS,
```
Comment (lipusz) [inline modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/utils/data.js:127]: @oliv-yu  We do have all the OOTB elements already available on master if this is what you meant here. :thinking:

Comment (lipusz) [reply modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/utils/data.js:127]: Or this this TODO for addressing the remaining work mentioned here? https://issues.liferay.com/browse/LPS-141101?focusedCommentId=2569876&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2569876

Comment (oliv-yu) [reply modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/utils/data.js:127]: It's that ticket, yes! We keep a copy of the baseline ootb elements on frontend for the "Add Blueprint" functionality.

Comment (lipusz) [reply modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/utils/data.js:127]: I see, thanks @oliv-yu ! :-)

**Pull request closed without merging**